### ISSUE TITLE
AOS specific config in ztp-cumulus.sh

### ISSUE THREAD

**Issue by bobbywatson3** [Apstra/aeon-ztps#169]
AOS specific configuration is in the ztp-cumulus.sh script that is called when cumulus-provision-url in DHCP options. This should be moved to the finally script in aos-aeonztps-addons.

```bash
function enable_mgmt_vrf(){
    if is_vrf_aware; then
            cat > /etc/network/interfaces << EOF
auto mgmt
iface mgmt
    address 127.0.0.1/8
    vrf-table auto
    post-up sudo service aos start

auto eth0
iface eth0 inet dhcp
    vrf mgmt
EOF
       ifreload -a
       ifdown eth0
       ifup eth0
    else
       install_vrf_deb
    fi
}
```

**Comment by jeremyschulman** [comment]
@bobbywatson3 - I am ok with this; but let's sync up offline re: some other changes proposed before we make this happen.  Thank you.

**Comment by bobbywatson3** [comment]
Resolved in #207
